usr/share/rear/lib/layout-functions.sh

```diff
@@ -590,3 +587,27 @@ is_disk_a_pv() {
 function is_multipath_path {
     [ "$1" ] && type multipath &>/dev/null && multipath -c /dev/$1 &>/dev/null
 }
+
+# retry_command () is binded with REAR_SLEEP_DELAY and REAR_MAX_RETRIES.
+# This function will do maximum of REAR_MAX_RETRIES command execution
+# and will sleep REAR_SLEEP_DELAY after each unsuccessful command execution.
+# Function returns command result as soon as it succeeded.
+function retry_command () {
+    command="$1"
+
+    for pass in trial verification ; do
```

Comment:
Actually this is not from my had ;-). Rationale is explained here in @jsmeix https://github.com/rear/rear/issues/1370#issuecomment-303046386.

In general _trial_ and _verification_ don't need to be define anywhere, they only purpose is to have checks run twice, just to be sure.
(Hope I've understood this correctly).

V.